package.json

```diff
@@ -73,7 +73,7 @@
   },
   "dependencies": {
     "await-semaphore": "^0.1.3",
-    "eth-contract-metadata": "github:estebanmino/eth-contract-metadata#e307359ca9ea6be4ded6ac58ac7e16f192ae4e2a",
+    "eth-contract-metadata": "github:estebanmino/eth-contract-metadata#d5e4e6377b0c0bcb5c33b9a4523aeba1f8d78d1e",
```

Comment:
We should consider using https://github.com/estebanmino/eth-contract-metadata/archive/d5e4e6377b0c0bcb5c33b9a4523aeba1f8d78d1e.zip here

Reply:
I think that might prevent Git-related issues from npm in the future

Reply:
Candidly, I thought tarballs and commit references behaved mostly the same under-the-hood: they're fetched from GH and optionally unzipped. Does using an archive have other benefits or behave differently?

Reply:
Well yes, technically they're the same, but I've seen a lot more Git-related errors than tarball-related errors from npm installs. npm also rewrites the `package-lock.json` file incessantly, often switching back and forth from hashes to no hashes (see also the diff on this PR). I think were they are the same "end result", using the tarball is less headache for users of the lockfile (and maybe shrinkwraps?).

Reply:
e.g. this is very common on existing lockfiles

<img width="1010" alt="" src="https://user-images.githubusercontent.com/1623628/54710372-79760c80-4b2a-11e9-8100-2024e86dab25.png">